Fix compile, don't narrow unsigned int to int.

Path of the diff has been adjusted to reflect upstream file moves.
From upstream revision 11603.

diff -r cf80fb39af0b -r 3d88c63d65bb src/rendering/SoOffscreenRenderer.cpp
--- src/misc/SoOffscreenRenderer.cpp	Tue Sep 26 09:47:59 2017 +0200
+++ src/misc/SoOffscreenRenderer.cpp	Tue Sep 26 09:49:30 2017 +0200
@@ -432,7 +432,7 @@
   size_t bufferbytesize;
 
   CoinOffscreenGLCanvas glcanvas;
-  unsigned int glcanvassize[2];
+  int glcanvassize[2];
 
   int numsubscreens[2];
   // The subscreen size of the current tile. (Less than max if it's a
@@ -1736,9 +1736,8 @@
   if (renderaction->getNumPasses() > 1) {
     SbVec3f jittervec;
     SbMatrix m;
-    const int vpsize[2] = { this->glcanvassize[0], this->glcanvassize[1] };
     coin_viewvolume_jitter(renderaction->getNumPasses(), renderaction->getCurPass(),
-                           vpsize, (float *)jittervec.getValue());
+                           this->glcanvassize, (float *)jittervec.getValue());
     m.setTranslate(jittervec);
     proj.multRight(m);
   }
